crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -807,6 +820,41 @@ impl VisitAstPath for Analyzer<'_> {
         }
     }
 
+    fn visit_new_expr<'ast: 'r, 'r>(
+        &mut self,
+        new_expr: &'ast NewExpr,
+        ast_path: &mut AstNodePath<AstParentNodeRef<'r>>,
+    ) {
+        // new URL("path", import.meta.url)
+        if let box Expr::Ident(ref callee) = &new_expr.callee {
+            if &*callee.sym == "URL" && is_unresolved(callee, self.eval_context.unresolved_mark) {
```

Comment:
I actually had essentially that in an earlier impl (removed in https://github.com/vercel/turbo/pull/2944/commits/77a84ae7bdcc35b29672608e425d2bfcb49a187b and https://github.com/vercel/turbo/pull/2944/commits/38e99d39ad975d94a4472c851711b4860da6ed72). I changed it because I couldn't fine later eval time code that inserted references into the code, so this seemed more consistent.